stack_definition.json

```diff
@@ -636,6 +695,14 @@
         {
           "name": "secrets_manager_region",
           "value": "ref:../5a - Security Service - Secret Manager/outputs/secrets_manager_region"
+        },
+        {
+          "name": "elastic_instance_crn",
+          "value": "ref:./members/3b - Databases for Elasticsearch/outputs/crn"
```

Comment:
Not sure if that's a correct reference - other inputs use `ref:../<member name>/outputs/...`